controllers/index.js

```diff
@@ -1,17 +1,20 @@
 const paragraph = require("../public/javascripts/data/paragraph");
 
 
-// GET request
+// GET request /
 // displayAllParagraphs function returns an array of all paragraphs in database
 exports.index = function(req, res, next) {
 
-  res.render('index', { ps: paragraph.displayAllParagraphs() });
+  let currentUserName = res.locals.userName;
+  res.render('index', { ps: paragraph.displayAllParagraphs(), currentUserName: currentUserName })
 };
 
-// POST request
+// POST request /
 exports.add_paragraph = function(req, res, next) {
   
+  let userId = res.locals.userId;
```

Comment:
Here I'd use `req.session.user.id` since it's going to communicate with the back-end